front/src/Components/Chat/ChatCircleDiscussion.svelte

```diff
@@ -0,0 +1,183 @@
+<script lang="ts">
```

Comment:
@cesar-cardinale this file is used in `front/src` or already moved in `front/chat/src`?